daemon/src/routes_maker.rs

```diff
@@ -152,27 +146,25 @@ pub struct PromptAuthentication {
 pub async fn post_cfd_action(
     id: OrderId,
     action: CfdAction,
-    cfd_actor: &State<Maker>,
+    maker_actor_system: &State<Maker>,
```

Comment:
Nit: I think naming this `maker` would be expressive enough.

```suggestion
    maker: &State<Maker>,
```